03-viz.Rmd

```diff
@@ -1,40 +1,528 @@
+(3) To better understand this we plotted data from 5 experiments by Michelson in 1879, each with 20 trials, as histograms stacked on top of one another.
+The horizontal axis shows the accuracy of the measurements relative to the true speed of light as we know it today, expressed as a percentage.
+From this visualization you can see that most results had relative errors of at most 0.05%. You can also see that experiments 1 and 3 had
+measurements that were the farthest from the true value, and experiment 5 tended to provide the most consistently accurate result. (4) It would be
+worth further investigation into the differences between these experiments to see why they produced different result distributions.
+
+
+
+## Saving the visualization
+#### *Choose the right output format for your needs* {-#my-section}
+
+Just as there are many ways to store datasets, there are many ways to store visualizations and images.
+Which one you choose can depend on a number of factors, such as file size/type limitations 
+(e.g., if you are submitting your visualization as part of a conference paper or to a poster printing shop)
+and where it will be displayed (e.g., online, in a paper, on a poster, on a billboard, in talk slides).
+Generally speaking, images come in two flavours: *bitmap* (or *raster*) formats and *vector* (or *scalable graphics*) formats.
+
+
+**Bitmap / Raster** images are represented as a 2-D grid of square pixels, each with their own colour. Raster images are often *compressed* before storing so they take up less space. A compressed format is *lossy* if the image cannot be perfectly recreated when loading and displaying, with the hope that the change is not noticeable. *Lossless* formats, on the other hand, allow a perfect display of the original image.
+
+- *Common file types:* 
+    - [JPEG](https://en.wikipedia.org/wiki/JPEG) (`.jpg`, `.jpeg`): lossy, usually used for photographs 
+    - [PNG](https://en.wikipedia.org/wiki/Portable_Network_Graphics) (`.png`): lossless, usually used for plots / line drawings
+    - [BMP](https://en.wikipedia.org/wiki/BMP_file_format) (`.bmp`): lossless, raw image data, no compression (rarely used)
+    - [TIFF](https://en.wikipedia.org/wiki/TIFF) (`.tif`, `.tiff`): typically lossless, no compression, used mostly in graphic arts, publishing
+- *Free Editing software:* [GIMP](https://www.gimp.org/), [JS Paint](https://jspaint.app)
+
+**Vector / Scalable Graphics** images are represented as a collection of mathematical objects (lines, surfaces, shapes, curves). When the computer displays the image, it redraws all of the elements using their mathematical formulas.
+
+- *Common file types:* 
+    - [SVG](https://en.wikipedia.org/wiki/Scalable_Vector_Graphics) (`.svg`): general-purpose use 
+    - [EPS](https://en.wikipedia.org/wiki/Encapsulated_PostScript) (`.eps`), general-purpose use (rarely used)
+- *Free Editing software:* [Inkscape](https://inkscape.org/), [Boxy SVG](https://boxy-svg.com/)
```

Comment:
```suggestion
- *Open-source Editing software:* [Inkscape](https://inkscape.org/), [Boxy SVG](https://boxy-svg.com/)
```

Reply:
These aren't all open-source, but in the interest of supporting open-source software I've removed the two non-open projects.